framework/src/interface/packet.rs

```diff
@@ -60,6 +60,22 @@ fn reference_mbuf(mbuf: *mut MBuf) {
     unsafe { (*mbuf).reference() };
 }
 
+impl<T, M> Clone for Packet<T, M>
```

Comment:
Probably need to make sure we have an implementation for when the other `packet_offset` configuration is used.